bin/ddg2logo

```diff
@@ -359,22 +230,20 @@ else:
 if options.config_file:
     try:
         with open(options.config_file) as f:
-            config = yaml.load(f, Loader=SafeLoader)
+            colors = yaml.load(f, Loader=SafeLoader)
     except:
         log.error("Couldn't open or parse config file. Exiting...")
         exit(1)
+elif options.color_scheme:
+    colors = options.color_scheme
 else:
-    # if config file doesn't exist, let's use a mock configuration
-    config = {'all' : {'residues' : ['A', 'C', 'D', 'E', 'F', 'G', 'H', 'I',
-                                     'K', 'L', 'M', 'N', 'P', 'Q', 'R', 'S',
-                                     'T', 'V', 'W', 'Y'],
-                       'color': [0., 0., 0.]}}
-
-new_letters = letters.copy()
-for key in config:
-    for aa in config[key]['residues']:
-        new_letters[aa] = color_letter(new_letters[aa], config[key]['color'])
-letters = new_letters
+    # if config file doesn't exist and a color scheme is not selected,
+    # let's use a mock configuration
+    colors = {'A':'black','C':'black','D':'black','E':'black','F':'black', 
```

Comment:
Maybe you need to say when colors are called something like "if args.colors = color_scheme, else: colors"

Reply:
we agreed on this as per the comment above